ci/utils/linux-common-runnner.sh

```diff
@@ -30,9 +27,7 @@ patch_apisix_code(){
     # ${1} apisix home dir
     VAR_APISIX_HOME="${VAR_CUR_HOME}/${1:-workbench}"
 
-    sed -ri -e "/make\s+ci-env-up/d" \
-      -e "/linux-ci-init-service.sh/d" \
-      "${VAR_APISIX_HOME}/ci/linux_openresty_common_runner.sh"
+    (cd $VAR_APISIX_HOME; patch -p1 < $VAR_CUR_HOME/ci/saml-auth.patch)
```

Comment:
I don't think patch is a good way because this depends on a specific version of APISIX.